src/audio/wav64.c

```diff
@@ -309,20 +321,21 @@ void wav64_open(wav64_t *wav, const char *fn) {
 	}	break;
 
 	case WAV64_FORMAT_OPUS: {
-		wav64_opus_init(wav, fh);
+		wav64_opus_init(wav);
 	}	break;
 	
 	default:
 		assertf(0, "wav64 %s: invalid format: %02x\n", fn, head.format);
 	}
 
-	dfs_close(fh);
+	close(fptr);
 }
 
 void wav64_play(wav64_t *wav, int ch)
 {
 	// Update the context pointer, so that we try to catch cases where the
 	// wav64_t instance was moved.
+	lseek(wav->currentFile, wav->baseOffset, SEEK_SET);
```

Comment:
This shouldn't be required if you implement the correct semantic when `seeking=True`.